.github/actions/install-chainweb-from-artifacts/action.yml

```
@@ -17,10 +17,10 @@ runs:
       run: |
         case "${{ inputs.os }}" in
         ubuntu*)
-          sudo apt-get install -y librocksdb-dev z3
+          sudo apt-get install -y z3 build-essential libsnappy-dev libgflags-dev libbz2-dev
           ;;
         mac*)
-          brew update && brew install rocksdb z3 || true
+          brew update && brew install gflags rocksdb llvm snappy bzip2 z3 gnu-tar || true
```

This action installs chainweb-node from the artifacts. Why are the build tools and development libraries needed?

```suggestion
          brew update && brew install gflags gnu-tar || true
```